src/building/suggested.md

```diff
@@ -175,6 +175,35 @@ You can also use `--keep-stage 1` when running tests. Something like this:
 - Initial test run: `./x.py test tests/ui`
 - Subsequent test run: `./x.py test tests/ui --keep-stage 1`
 
+## Incremental builds
+
+Incremental compilation is enabled by default in almost all of the config profiles
+under the `src/bootstrap/defaults`.
+
+In case if it's not enabled, you can configure rustbuild to use incremental
+compilation with the `--incremental` flag:
+
+```sh
+$ ./x.py build --incremental
+```
+
+If you want to enable it once for all, you can apply the following configuration
+in your `config.toml`:
+
+```toml
+[rust]
+incremental=true
+```
+
+Incremental compilation will store compilation artifacts
+in `build/<host>/stage0-incremental`. Note that we only use incremental
+compilation for the stage0 -> stage1 compilation -- this is because
+the stage1 compiler is changing, and we don't try to cache and reuse
+incremental artifacts across different versions of the compiler.
```

Comment:
This isn't quite what I meant, sorry - I was suggesting that you cut and paste https://rustc-dev-guide.rust-lang.org/tests/running.html#using-incremental-compilation, i.e. use exactly the wording from that section here and delete it in `running.md`. This new text has some inaccuracies I would rather not try to fix line-by-line.